### DISCUSSION TITLE
Need help? Start here!

### DISCUSSION THREAD

**Discussion by SebiAi** [SebiAi/custom-nothing-glyph-tools#14] [Q&A]
This is the perfect place if you need help or have questions regarding this project!

Create a new discussion here: https://github.com/SebiAi/custom-nothing-glyph-tools/discussions/new/choose

**Comment by un4tuner** [comment]
Did everything by instruction. Checked codecs and stuff...
Glyph composer says that ogg file is in ogx format and does not want to import them. =(

**Reply by SebiAi** [reply]
Did you use the [Troubleshooting](https://github.com/SebiAi/custom-nothing-glyph-tools#interrobang-troubleshooting) chapter of the README? If yes, what is the output of ffprobe?

How did you transfer the file to your phone? Google Drive is known to modify the file extension - use another way, like via USB, Discord, WhatsApp, Signal etc.

**Reply by un4tuner** [reply]
Damn I'm stupid! It was Google Drive. Uploaded directly - works like a charm!! Thank you!
